src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -2869,6 +2869,33 @@ public static unsafe void ConstructorsTest_InvalidUTF8()
             }
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
```

Comment:
Sorry, I got it wrong - it should be without the tilda:

` [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]`